js/script.js

```diff
@@ -14,10 +13,13 @@ var RandomDeclutter = RandomDeclutter || {};
 		_containerBefore: 4,
 		_containerCurrent: 1,
 		_containerAfter: 2,
+		_containerActive: 'div.active div.element-preview',
```

Comment:
The `div` parts are not necessary, the element can simply be identified by the class.